test.rb

```diff
@@ -0,0 +1 @@
+path = Pathname.new("foo/bar/baz")
```

Comment:
__[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `path`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->